etrago/cluster/networkclustering.py

```diff
@@ -261,6 +261,7 @@ def strategies_one_ports():
             "e_nom": np.sum,
             "e_nom_min": np.sum,
             "e_nom_max": np.sum,
+            "e_initial": np.mean,
```

Comment:
Energies are aggregated, so I think you should use `np.sum` instead of `np.mean`. What do you think?

Reply:
Agree. I assumed wrongly that "e_initial" was in p.u. as well as "e_nom_min" and "e_nom_max". It was corrected in b2beae8b3b28f1e2c11918058da5b524bfc779fd.